src/bonuses.cpp

```diff
@@ -183,6 +219,51 @@ float bonus_container::get_mult( const Character &u, affected_stat stat ) const
     return get_mult( u, stat, DT_NULL );
 }
 
+std::string bonus_container::get_description() const
+{
+    std::stringstream dump;
+    for( const auto &boni : bonuses_mult ) {
+        std::string type = string_from_affected_stat( boni.first.get_stat() );
+
+        if( needs_damage_type( boni.first.get_stat() ) ) {
+            type = name_by_dt( boni.first.get_damage_type() ) + " " + type;
+        }
+
+        for( const auto &sf : boni.second ) {
+            dump << string_format( "%s: <stat>%d%%</stat>", type, static_cast<int>( sf.scale * 100 ) );
+
+            if( sf.stat ) {
+                dump << _( " of " ) << string_from_scaling_stat( sf.stat );
```

Comment:
Could you please add a translation comment for this? Here and below.

Reply:
Fixed